admin_guide/manage_nodes.adoc

```diff
@@ -415,32 +415,34 @@ anti-affinity can be set to either required or preferred.
 
 Using the `*docker-registry*` pod as an example, the first step in enabling
 this feature is to set the `*scheduler.alpha.kubernetes.io/affinity*` on the
-pod. Since this pod uses a deployment configuration, the most appropriate
-place to add the annotation is to the pod template's metadata.
-
-----
-$ oc edit dc/docker-registry -o yaml
-
-...
-  template:
-    metadata:
-      annotations:
-        scheduler.alpha.kubernetes.io/affinity: |
-          {
-            "podAntiAffinity": {
-              "requiredDuringSchedulingIgnoredDuringExecution": [{
-                "labelSelector": {
-                  "matchExpressions": [{
-                    "key": "docker-registry",
-                    "operator": "In",
-                    "values":["default"]
-                  }]
-                },
-                "topologyKey": "kubernetes.io/hostname"
-              }]
-            }
-          }
-----
+pod. 
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: with-pod-antiaffinity
+spec:
+  affinity:
+    podAntiAffinity: <1>
+      preferredDuringSchedulingIgnoredDuringExecution: <2>
+      - weight: 100 <3>
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: docker-registry <4>
+              operator: In <5>
+              values: 
+              - default
+          topologyKey: kubernetes.io/hostname
+----
+
+<1> Stanza to configure pod anti-affinity.
+<2> Defines a preferred rule.
+<3> Specifies a weight for a preferred rule. The node that with highest weight is preferred.
```

Comment:
"The node that with highest weight is preferred." doesn't make sense.
I haven't read any of the context, but I think it should say something like "The pod runs on the node with the highest weight" or "The pod applies the rule with the largest weight."

Reply:
@kalexand-rh I think the meaning is right. Rules can be required to be followed or followed unless it is not possible (preferred). This setting is only for a preferred rule. If the wording can be better, let me know!

Reply:
After our chat, I think we should revisit this after 3.10

s/that with/with the